pkg/manifests/reader.go

```diff
@@ -92,6 +93,6 @@ func (r *Reader) ReadCharts(eksaVersion string) ([]releasev1.Image, error) {
 	return bundles.Charts(bundle), nil
 }
 
-func (r *Reader) ReadChartsFromBundles(b *releasev1.Bundles) []releasev1.Image {
+func (r *Reader) ReadChartsFromBundles(ctx context.Context, b *releasev1.Bundles) []releasev1.Image {
```

Comment:
nit: let's create another method that doesn't need the context and leave this just to implement the interface from download/import

Reply:
Discussed offline, this will be included in a follow up PR.